service/src/main/java/org/cbioportal/service/impl/GenePanelServiceImpl.java

```diff
@@ -104,6 +104,9 @@ public GenePanel getGenePanel(String genePanelId) throws GenePanelNotFoundExcept
 
         molecularProfileService.getMolecularProfile(molecularProfileId);
         List<String> sampleIds = sampleListRepository.getAllSampleIdsInSampleList(sampleListId);
+        if (sampleIds.isEmpty()) {
+            return new ArrayList<>();
```

Comment:
Why not `Collections.emptyList()`?

Reply:
it's not type safe, but I'll change to Collections.<GenePanelData>emptyList()